front-end/src/environments/environment.cloud.gov.dev.ts

```diff
@@ -0,0 +1,24 @@
+// This file can be replaced during build by using the `fileReplacements` array.
+// `ng build --configuration=cloud.gov.dev` replaces `environment.ts` with `environment.cloud.gov.dev.ts`.
+// The list of file replacements can be found in `angular.json`.
+
+export const environment = {
+    production: false,
+    name: 'development',
+    apiUrl: 'https://fecfile-web-api-dev.app.cloud.gov/api/v1',
+    appTitle: 'FECfile',
+    validateSuccess: 'All required fields have passed validation.',
+    awsRegion: 'us-east-1',
```

Comment:
Are these AWS variables needed for cloud.gov? If not let's remove them from here and the other environment.*.ts files

Reply:
Just did a test and "ng server" generates errors is we remove awsRegion or awsIdentityPoolId so I am going to leave them in place.